### PR TITLE
Expanding on triangle time blending doc strings.

### PR DESCRIPTION
**Pull request by Kat-90**
Closes https://github.com/metoppv/mo-blue-team/issues/115

Description:

The doc strings relating to the triangle time blending functionality had some errors in and needed updating so that they were consistent across the multiple related files.

Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

File: improver/blending/blend_across_adjacent_points.py [hunk 1]
```diff
@@ -44,11 +44,14 @@
...
     """
-    Apply a Weighted blend to a coordinate, using triangular weights at each
-    point in the coordinate. Returns a cube with the same coordinates as the
-    input cube, with each point in the coordinate of interest having been
-    blended with the adjacent points according to a triangular weighting
-    function of a specified width.
+    Applies a weighted blend to the data using a triangular weighting function
+    at each point in the dimension so that maximum weighting is applied to the
+    specified point, and weighting falls away at neighbouring points the
+    further they get until the specified width has been reached.
```
Comment (MoseleyS) [inline improver/blending/blend_across_adjacent_points.py:50]:
```suggestion
    Applies a weighted blend to the data using a triangular weighting function
    at each point in the specified dimension. The maximum weighting is applied
    to the specified point, and weighting decreases linearly for neighbouring
    points to zero at the specified triangle width.
```

**Review by MoseleyS — CHANGES_REQUESTED** [review]
Looks good, but here's a small suggestion.

**Review by MoseleyS — APPROVED** [review]

File: improver/blending/weights.py [hunk 2]
```diff
@@ -755,10 +755,13 @@ def triangular_weights(
             midpoint:
                 The centre point of the triangular function.
             width:
-                The width of the triangular function from the centre point.
+                The width from the triangle’s centre point, in units of the units
+                argument, which will determine the triangular weighting function
```
Comment (bayliffe) [inline improver/blending/weights.py:759]: > The width from the triangle’s centre point, in units of the **_plugin's_** units argument, which will determine the triangular weighting function....


This function doesn't have a units argument, so it's probably worth pointing the reader up to the plugin level.

**Review by bayliffe — COMMENTED** [review]
Looks much better Kat, thanks. One teeny tiny comment.

**Review by bayliffe — APPROVED** [review]

**Pull request merged**